Luqman011/PropertyPro-Lite
# Merges UI to develop

**what does this PR do?**
Merges UI to Develop

**Description of a task to be completed**
merges UI to develop
add new changes to develop

**how should this be manually tested?**

open GitHub
navigate to the relevant repository
check the branches
go to the develop branch
